### DISCUSSION TITLE
How to distinguish GWL timeslice runs

### DISCUSSION THREAD

**Discussion by larsbuntemeyer** [WCRP-CORDEX/discuss#17] [Q&A] [answered]
@jesusff @gnikulin 
I just realized that we have some projects that make up their own `experiment_id`, e.g., based on global warming levels of the driving GCM. Also in Impetus there are some experiments running several time slices for different GWL from the same GCM. These experiments probably make up some `experiment_id` like, `ssp370-GWL2K` or `ssp370-GWL3K` to distinguish time slices as different experiments. I am not sure if these kind of experiments are supposed to be published in ESGF (i know that at least in NUKLEUS this is expected?). In that case these experiments could not be distinguished in the DRS because it contains the same `driving_experiment_id`. Should we use an `experiment_id` instead? I vaguely remember already having discussed this at sometime, urghhh...

EDIT: Maybe we wanted to distinguish this as different `version_realization`?

**Comment by jesusff** [comment, marked as answer]
I wonder if these are just different realizations. GWLs only mean downscaling a particular period. E.g. in I4C there are groups that will run a full transient to cover all GWLs in one go. The main difference for GWL experiments is that the output will  only be available for a subperiod and, most importantly, that there could be different subperiods (GWL periods) with exactly the same filename. However, if these [slices are carried out independently](https://doi.org/10.1007/s00382-022-06560-2), these would just be different realizations done with the same model and driven by the same boundaries. Therefore, they could be labeled v1-r1, v1-r2, etc. and the `version_realization_info` be used to explain that e.g. _Realization r2 covers GWL +3K of NorESM2-MM r1i1p1f1, corresponding to period 2081-2100_.

**Comment by larsbuntemeyer** [comment]
Ok, agreed! That makes sense and would also significantly reduce cluttering the `experiment_id` with tons of different realization or GWL info!

**Comment by gnikulin** [comment]
May be I'm missing something here but downscaling periods with different GWLs for GCMs is simply downscaling of a number of time slices. The scenario `driving_experiment_id` is the always same. From this side there is no problem to publish such simulations in ESGF. 

I didn't get why file names should be the same in such experiments ?

> The main difference for GWL experiments is that the output will only be available for a subperiod and, most importantly, that there could be different subperiods (GWL periods) with exactly the same filename. 

I wouldn't touch `version_realization_info`, it's not about realization.

**Comment by larsbuntemeyer** [comment]
I think, they are different realizations in the sense that these runs start with different intial conditions at the beginning of the GWL which would give different results than, e.g., a transient run that simply covers the whole 21st century. GWL runs could be overlapping as well so you would have duplicated timesteps if you don't disitinguish those simulations somehow...

**Comment by gnikulin** [comment]
OK I see and would say that this is a specific experiment which in general doesn't fit the CORDEX-CMIP6 experiment design and cannot be published under the CORDEX-CMIP6 project in ESGF. Of course, no problem to create another project in ESGF  to publish such simulations.

**Comment by jesusff** [comment]
Yes, there will also be FPSs using this setup, especially those running CP simulations. They can be published in a different ESGF project, but the configuration can be the same, as there are no new elements. In this sense, they could also simply go under a different activity_id (different from DD), as already planned.

**Comment by gnikulin** [comment]
If it's a CORDEX FPS, of course, simulations can be published under the CORDEX-CMIP6 project with a different activity. Keeping the same number of the DRS elements as in DD is preferable, makes life easier :-).
